copier/main.py

```diff
@@ -320,7 +321,10 @@ def _render_allowed(
         if dst_relpath != Path(".") and self.match_exclude(dst_relpath):
             return False
         try:
-            previous_content = dst_abspath.read_bytes()
+            if is_symlink:
+                previous_content = bytes(readlink(dst_abspath))
```

Comment:
Seems less error-prone:

```suggestion
                previous_content = readlink(dst_abspath)
```